packages/react-native-views/src/components/TutorialView/Illustration.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+export default () => (
+  <svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" height="220px" viewBox="0 0 858.756 581.934">
```

Comment:
We loose the fact that this is a svg file (with all the advantages, such as the ability to preview and edit it as such) and it becomes hard to tell if the native and non native file are the same or not.
Wouldn't it be better to have an actual svg file and manipulate it?

Reply:
It's complicated because svg files are not loaded the same way in every projects. Between the default Metro config or the default Webpack config there is a huge difference. Loading svg like this ensure that developers using SDK will not have to configure another loader to load SVGs.

Reply:
Ok. If it's the same file then would it be possible to make it available as a string and then parse it differently depending on if it's native or not (rather than having it twice)